src/wymeditor/editor/keyboard.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+WYMeditor.Keyboard = function (wym) {
+    var keyboard = this;
+
+    keyboard._wym = wym;
+
+    keyboard.combokeys = new WYMeditor.EXTERNAL_MODULES.Combokeys(wym._doc);
+
+    return keyboard;
+};
+
+WYMeditor.Keyboard.DEFAULT_KEYBOARD_SHORTCUTS = [
+    {
+        combo: "mod+b",
+        cb: function () {
+            var wym = this;
+            wym.exec(WYMeditor.EXEC_COMMANDS.BOLD);
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    },
+    {
+        combo: "mod+i",
+        cb: function () {
+            var wym = this;
+            wym.exec(WYMeditor.EXEC_COMMANDS.ITALIC);
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    },
+    {
+        combo: "mod+k",
+        cb: function () {
+            var wym = this;
+            wym.dialog(WYMeditor.DIALOGS.CreateLink);
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    },
+    {
+        combo: "tab",
```

Comment:
Whoa. Nice find!

Does this fix break anything with the list plugin's tab handling? I'm not exactly sure how multiple combos bound to the same key are handled.